sdk/event.proto

```diff
@@ -39,12 +38,16 @@ message PushEvent {
     uint32 commits = 4;
     // Commits is the number of distinct commits in the push.
     uint32 distinct_commits = 5;
+    // Configuration contains any configuration related to specific analyzer
+    google.protobuf.Struct configuration = 6 [(gogoproto.nullable) = false];
 
     CommitRevision commit_revision = 7 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
 }
 
 // ReviewEvent represents a Review (PullRequest in case of Github) being created or updated.
 message ReviewEvent {
+    option (gogoproto.protosizer) = false;
```

Comment:
As you say, it takes time to figure it out, so might be a good idea to leave a bread-crumb i.e a comment on why is this required. Wdyt?